lib/elixir/lib/enum.ex

```diff
@@ -1549,6 +1571,20 @@ defmodule Enum do
     { :lists.reverse(list_acc), acc }
   end
 
+  ## member?
+
+  def do_member?(:stop, _, _) do
```

Comment:
Yes, Please send a PR of they are still public!

On Wednesday, March 2, 2016, Oskar Boethius Lissheim <
notifications@github.com> wrote:

> In lib/elixir/lib/enum.ex
> <https://github.com/elixir-lang/elixir/pull/992#discussion_r54798954>:
>
> > @@ -1549,6 +1571,20 @@ defmodule Enum do
> >      { :lists.reverse(list_acc), acc }
> >    end
> >
> > +  ## member?
> > +
> > +  def do_member?(:stop, _, _) do
>
> Shouldn't these funcs be defined as private (defp) like the others?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/elixir-lang/elixir/pull/992/files#r54798954>.
>


-- 


*José Valimwww.plataformatec.com.br
<http://www.plataformatec.com.br/>Founder and Director of R&D*